scripts/smoke.mjs

```
@@ -0,0 +1,77 @@
+const apiBase = process.env.API_BASE ?? "http://localhost:7000";
```

ESLint

[error] 'process' is not defined.

(no-undef)